File: app/src/main/java/com/zillennium/utswap/models/ListMenu/ListMenuAdapter.java
package com.zillennium.utswap.models.ListMenu;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ArrayAdapter;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.zillennium.utswap.R;

import java.util.ArrayList;

public class ListMenuAdapter extends ArrayAdapter<ListMenu> {
    public ListMenuAdapter(Context context, ArrayList<ListMenu> accountArrayList){
        super(context, R.layout.item_list_setting_menu, accountArrayList);
    }

    @NonNull
    @Override
    public View getView(int position, @Nullable View convertView, @NonNull ViewGroup parent) {

        ListMenu account = getItem(position);

        if(convertView == null){
            convertView = LayoutInflater.from(getContext()).inflate(R.layout.item_list_setting_menu, parent, false);
        }

        TextView txt_title = convertView.findViewById(R.id.txt_title);
        ImageView iv_icon = convertView.findViewById(R.id.iv_icon);

        txt_title.setText(account.title);
        iv_icon.setImageResource(account.icon);

        return convertView;
//        return super.getView(position, convertView, parent);
    }
}

File: app/src/main/java/com/zillennium/utswap/models/ListMenu/ListMenuAdapter.kt
package com.zillennium.utswap.models.ListMenu

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ArrayAdapter
import android.widget.ImageView
import android.widget.TextView
import com.zillennium.utswap.R


class ListMenuAdapter(context: Context?, accountArrayList: ArrayList<ListMenu>) :
    ArrayAdapter<ListMenu?>(
        context!!, R.layout.item_list_setting_menu,
        accountArrayList!! as List<ListMenu?>
    ) {
    override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
        var convertView = convertView
        val account = getItem(position)
        if (convertView == null) {
            convertView =
                LayoutInflater.from(context).inflate(R.layout.item_list_setting_menu, parent, false)
        }
        val txtTitle = convertView!!.findViewById<TextView>(R.id.txt_title)
        val ivIcon = convertView.findViewById<ImageView>(R.id.iv_icon)
        txtTitle.text = account!!.title
        ivIcon.setImageResource(account.icon)
        return convertView
        //        return super.getView(position, convertView, parent);
    }
}